### ISSUE TITLE
AoT fails when app.routes.ts is in the root src directory

### ISSUE THREAD

**Issue by isaacplmann** [brandonroberts/angular2-router-loader#6]
If your app structure is like this:

```
src/
- app.module.ts
- app.routes.ts
- compiled/
- features/
  - lazy/
    - lazy.module.ts
```

And `app.routes.ts` references `lazy.module.ts` like this:

```
{
  loadChildren: './features/lazy/lazy.module#LazyModule',
  path: 'lazy',
},
```

And `tsconfig.json` has this setting:

```
  "angularCompilerOptions": {
    "genDir": "src/compiled",
    "skipMetadataEmit" : true
  }
```

And the `angular2-router-loader` line in webpack config is this:
`'angular2-router-loader?loader=system&genDir=src/compiled/src&aot=true'`

Then you get an error stating: (from memory)
`Could not find 'features/lazy/lazy.module.ts' in directory 'correct/path/to/src'`

This is a pretty nonsensical error, since if you put those two paths together, you get the right path to `lazy.module.ts`.  A workaround is to move everything except the `compiled` folder into a new `src/app` folder.  Perhaps this could be fixed by prepending the relative path with './'?

**Comment by brandonroberts** [comment]
Yea, it seems that it doesn't prepend the relative path with a `./` if its at the same level of the `genDir`. I'll submit a fix and get it published.

Thanks!

**Comment by brandonroberts** [comment]
Fixed via https://github.com/brandonroberts/angular2-router-loader/commit/01fb92e28dbf42ee35629cb8a6708bfa5590aa5c

Published as `0.3.1`
